1-exercises/A-accessing-values/exercise2.js

```diff
@@ -17,7 +17,7 @@ let capitalCities = {
 */
 
 let myCountry = "UnitedKingdom";
-let myCapitalCity; // complete the code
+let myCapitalCity=capitalCities["UnitedKingdom"]; // complete the code
```

Comment:
This works in this case. But it is important to note that you can also write `let myCapitalCity=capitalCities[myCountry];`, which would allow you to access other properties in the object if you change the variable myCountry.